backend/devflow-interfaces/src/main/kotlin/com/devflow/interfaces/analytics/OrganizationAnalyticsController.kt

```
@@ -233,45 +233,65 @@ class OrganizationAnalyticsController(
+                else -> false
             }
 
-            ResponseEntity.ok(response)
+            // 4. Response DTO로 변환
+            val responses = repositoryAnalyticsList.map { analytics ->
+                repositoryAnalyticsMapper.toResponse(analytics, analyticsPeriod, isRealTime)
+            }
+
+            ResponseEntity.ok(responses)
+
         } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)